test/clj/game_test/cards/events.clj

```diff
@@ -1631,6 +1631,41 @@
     (click-prompt state :runner "HQ")
     (run-successful state)))
 
+(deftest labor-rights
+  ;; Labor Rights - trash 3 cards, shuffle 3 cards from heap->stack, draw 1 card, rfg Labor Rights
+  (testing "Basic behavior"
+    (do-game
+      (new-game {:runner {:deck ["Labor Rights" "Sure Gamble" "Lawyer Up" "Knifed"]}})
+      (take-credits state :corp)
+      (core/move state :runner (find-card "Sure Gamble" (:hand (get-runner))) :deck)
```

Comment:
It should be possible to write these lines as `:deck ["Sure Gamble" "Lawyer Up" "Knifed"] :hand ["Labor Rights"]` in the `new-game` map.